jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Environment.java

```diff
@@ -46,10 +50,22 @@ static Environment set(Environment environment)
         return Named.__environments.put(environment.getName(), environment);
     }
 
+    /**
+     * @return The case insensitvie name of the environment.
+     */
     String getName();
```

Comment:
Perhaps add, like other classes, an `is(String name)` method that does the lowercase equals?